compose/cli-command.md

```diff
@@ -1,7 +1,7 @@
 ---
 description: Compose V2 in the Docker CLI
-keywords: compose, V2, beta, Tech preview
-title: Compose V2 beta
+keywords: compose, V2, release candidate, Tech preview
+title: Compose V2 release candidate 1
```

Comment:
@ulyssessouza This is updated as part of https://github.com/docker/docker.github.io/pull/13318/files. Could you remove this from your PR?